### DISCUSSION TITLE
Wordlist

### DISCUSSION THREAD

**Discussion by ablito19** [pablosnt/rekono#36] [Help] [answered]
Hello, why in Wordlist page, i have empty size for every line. 
Do i need add manually files or it already done ?

**Comment by pablosnt** [comment]
This is because the default wordlists are pointing to default wordlists in Kali Linux OS. The wordlists size is stored in database when the wordlist is created or modified, so Rekono can't store its size by default because it could change. You can use wordlist without size info anyway, and if you need it, you can save it manually on database. I'm opened to suggestions about this topic

**Comment by pablosnt** [comment, marked as answer]
Since the Rekono version `1.4.0`, the size of default wordlists is saved automatically.
